front/src/applications/stdcm/utils/formatStdcmConfV2.ts

```diff
@@ -29,6 +29,7 @@ type ValidStdcmConfig = {
   totalMass?: number;
   totalLength?: number;
   maxSpeed?: number;
+  towedRollingStockID?: number;
```

Comment:
Same than in the back, maybe putting this field next to `rollingStockId` would make sense (semantically because their close, but also in the UI, they're close).